rest-server/src/main/resources/nubesgen/bicep/modules/app-service/app-service.bicep.mustache

```diff
@@ -131,11 +131,11 @@ resource appServiceApp 'Microsoft.Web/sites@2020-12-01' = {
         }
         {
           name: 'DOCKER_REGISTRY_SERVER_USERNAME'
-          value: '${containerRegistry.name}'
+          value: containerRegistry.name
         }
         {
           name: 'DOCKER_REGISTRY_SERVER_PASSWORD'
-          value: listCredentials(containerRegistry.id, containerRegistry.apiVersion).passwords[0].value
+          value: containerRegistry.listCredentials().passwords[0].value
```

Comment:
I didn't test this syntax with the NubesGen output, but it's same as what I use here: https://github.com/pamelafox/flask-charts-api-container-app/blob/main/infra/core/host/container-app.bicep#L38

Reply:
According to the doc, it shuold work. I'll test with NubesGen ;)